extensions/assets-manager/AssetsManager.cpp

```diff
@@ -197,7 +197,6 @@ std::string AssetsManager::keyOfDownloadedVersion() const
 bool AssetsManager::checkUpdate()
 {
     if (_versionFileUrl.size() == 0 || _isDownloading) return false;
-    
```

Comment:
Having a space line makes codes more clear. So i don't think need to remove it.

Reply:
recovered